package.json

```diff
@@ -19,19 +19,16 @@
   "bugs": {
     "url": "https://github.com/IFS49F/poker/issues"
   },
-  "dependencies": {
-    "bootstrap": "^3.3.7",
-    "jquery": "^3.1.1",
-    "js-cookie": "^2.1.3",
-    "leancloud-storage": "^1.5.3",
-    "requirejs": "^2.3.2"
-  },
   "license": "MIT",
   "devDependencies": {
+    "bootstrap": "^3.3.7",
```

Comment:
Why we need to move all dependencies into devDependencies?

I think `dependencies` is used for production, which means we can run `npm install --save` to install necessary packages in `dependencies` only. During development, if we only run `npm install`, there is no difference.

It's not a big problem, but I'd like to distinguish them in repo.